tests/scripts/vg_wrap/valgrind

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Wrap valgrind to run tests from a temp directory. This allows tests to be run in parallel without
+# corrupting each other's temp files.
+
```

Comment:
In the full config (more generally, when `MBEDTLS_ENTROPY_NV_SEED` is enabled), you need to create a `seedfile`.

Reply:
Now it's using #8294 this isn't an issue (it will use the one in `tests`)